WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -74,6 +76,26 @@ + (NSInteger)mjpegServerPort
   return DefaultMjpegServerPort;
 }
 
++ (NSInteger)mjpegScalingFactor
```

Comment:
I'd rather set these via settings interface, like we set the FPS

Reply:
For settings we need a separate request to WDA right?